### PR TITLE
Updated to use oxyplot core package to v2.1

### PR DESCRIPTION
**Pull request by saskathex**
Fixes # .

### Checklist

- [x] I have included examples or tests
- [x] I have updated the change log
- [x] I am listed in the CONTRIBUTORS file
- [x] I have cleaned up the commit history (use rebase and squash)

### Changes proposed in this pull request:

- Bumped oxyplot core package to 2.1
- Update projects to use latests .net framework 4.8
- Using GtkSharp (for gtk3 bindings) instead less popular gtk-sharp-3 package

@oxyplot/admins

### CODE REVIEW

**Comment by saskathex** [comment]
I am working with my private .net50 oxyplot.gtksharp sources for the last year in my private project. After upgrading to oxyplot.core 2.1 I released there were some issue. Fixed them and somehow managed to create my first pull request. As you propably realized I had some issues with the to me unknown AppVeyor ci pipeline.

@VisualMelon: Go ahead and merge #10. I can then base a pull request on that source base. The multitargets nupkg with .net452, net48 and net50 is also no problem.

File: Source/OxyPlot.GtkSharp.Shared/PlotView.cs [hunk 1]
```diff
@@ -285,20 +285,28 @@ public void ShowTracker(TrackerHitResult data)
...
             this.trackerLabel.Parent.Visible = true;
             this.trackerLabel.Text = data.ToString();
+#if GTKSHARP3
+            int pw = trackerLabel.Parent.AllocatedWidth;
+            int ph = trackerLabel.Parent.AllocatedHeight;
+            int xPos = (int)data.Position.X - pw / 2;
+            int yPos = (int)data.Position.Y - ph;
+            xPos = Math.Max(0, Math.Min(xPos, this.Allocation.Width - pw));
+            yPos = Math.Max(0, Math.Min(yPos, this.Allocation.Height - ph));
+#else
```
Comment (VisualMelon) [inline Source/OxyPlot.GtkSharp.Shared/PlotView.cs:305]: What's the idea here?

File: Source/OxyPlot.GtkSharp.Shared/PlotView.cs [hunk 2]
```diff
@@ -285,20 +285,28 @@ public void ShowTracker(TrackerHitResult data)
                 // us to set the background color
                 Gtk.EventBox labelHolder = new Gtk.EventBox();
                 this.trackerLabel = new Gtk.Label();
-                this.trackerLabel.SetPadding(3, 3);
+                this.trackerLabel.Xpad = 3;
+                this.trackerLabel.Ypad = 3;
                 OxyColor bgColor = OxyColors.LightSkyBlue;
-                labelHolder.ModifyBg(StateType.Normal, new Gdk.Color(bgColor.R, bgColor.G, bgColor.B));
+                //labelHolder.ModifyBg(StateType.Normal, new Gdk.Color(bgColor.R, bgColor.G, bgColor.B));
```
Comment (VisualMelon) [inline Source/OxyPlot.GtkSharp.Shared/PlotView.cs:291]: Is this a regression?

File: Source/OxyPlot.GtkSharp.Shared/PlotView.cs (hunk 1 above)
Comment (VisualMelon) [reply Source/OxyPlot.GtkSharp.Shared/PlotView.cs:305]: @hol430 does this look like something we might want to include on-top of your work?

I'm tempted to get on and merge #16, and pick up the pieces in separate PR (e.g. retarget to 2.1 proper; retarget example library). I'm still not in a good position to test it, but I'm reassured seeing that your work here overlaps with #10.

Comment (VisualMelon) [reply Source/OxyPlot.GtkSharp.Shared/PlotView.cs:305]: @hol430 thanks so much for taking the time to look at this.

**Comment by VisualMelon** [comment]
Closing having merged #16

**Pull request closed without merging**